src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelRegulationLegend.tsx

```diff
@@ -36,12 +36,22 @@ import { getBrowserActiveGenomeId } from 'src/content/app/genome-browser/state/b
 import styles from '../TrackPanelList.scss';
 import trackPanelItemStyles from './TrackPanelItem.scss';
 import regulationStyles from 'src/content/app/genome-browser/components/drawer/drawer-views/regulation-legend/RegulationLegend.scss';
+import { useGetRegulationStatsQuery } from 'src/content/app/genome-browser/state/api/genomeBrowserApiSlice';
 
 const TrackPanelRegulationLegend = (props: { disabled: boolean }) => {
   const activeGenomeId = useAppSelector(getBrowserActiveGenomeId) as string;
   const dispatch = useDispatch();
   const { trackDrawerOpened, reportTrackPanelSectionToggled } =
     useGenomeBrowserAnalytics();
+  const { data: regulationStats } = useGetRegulationStatsQuery(activeGenomeId);
+
+  if (!regulationStats) {
+    return;
+  }
+  const availableRegulationLegends = regulationLegends.filter((legend) => {
```

Comment:
Mmm, there is something about the names that we missed last time.

A legend, if applied to an explanatory note next to an image, is the note as a whole. It is not just one colour/label combination; it is all of them. Thus, when this PR is using the word "legend" to refer to just a single colour square with text next to it, and calls them collectively by the plural form "legends", it is using this word incorrectly.

Notice that the component itself is called `...Legend`.

Could we possibly rename "legends" (the constants file and its contents) to "legend"; and change the variable name here to something different. `availableRegulatoryFeatures` would be nice.